tutorials/market/PyData_Demo_1.ipynb

```diff
@@ -0,0 +1,2455 @@
+{
```

Comment:
<p><code>copy()</code>  not necessary, reset_index will create a copy unless <code>inplace=True</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Good point, I'll drop that from the next version of this file.</p>